src/pages/home/ReportScreen.js

```diff
@@ -239,7 +238,6 @@ function ReportScreen({
                 policy={policy}
                 personalDetails={personalDetails}
                 isSingleTransactionView={isSingleTransactionView}
-                parentReportAction={parentReportAction}
```

Comment:
Turns out this component no longer references a `parentReportAction` prop, so I just removed it. The `parentReportAction` is only used in the `MoneyRequestHeader`.